src/main/java/com/hillel/items_exchange/service/impl/AdvertisementServiceImpl.java

```diff
@@ -199,4 +217,47 @@ private AdvertisementTitleDto buildAdvertisementTitle(Advertisement advertisemen
                 .ownerAvatar(advertisement.getUser().getAvatarImage())
                 .build();
     }
+
+    private AdvertisementDisplayDto buildAdvertisementDisplayDto(Advertisement advertisement) {
+        Location advLocation = advertisement.getLocation();
```

Comment:
I couldn't found where you use that variable...

Reply:
Changed